### PR TITLE
Pokemon game

### PR DESCRIPTION
**Pull request by ivannemeth**

### CODE REVIEW

**Review by thisKonrad — COMMENTED** [review]
Good Done!

File: css-basic2/css-basics_pokemon-page/css/styles.css [hunk 1]
```diff
@@ -5,11 +5,14 @@
...
+}
 /* Task 2: Let's work on some basic stylings that we want to apply to all elements of your code via inheritance. Which selector do we need for this? */
 
 /* add selector */
- {
+
+* {
   font-family: monospace;
   font-size: 1rem;
   line-height: 1.5;
```
Comment (emyjamalian) [inline css-basic2/css-basics_pokemon-page/css/styles.css:17]: to have a cleaner code it's better to have only one class definition. so could you please have it all in one place?

* {
box-sizing: border-box;
font-family: monospace;
  font-size: 1rem;
  line-height: 1.5;
  }

File: css-basic2/css-basics_pokemon-page/css/styles.css [hunk 2]
```diff
@@ -21,6 +24,9 @@
 /* Task 3: To set the line height to 1.5 is good for texts, because of accessibility concerns, but for the headings it seems too much. Let's change this by giving our headings a line height of 1.2: */
 
 /* your solution */
+h1, h2 {
```
Comment (emyjamalian) [inline css-basic2/css-basics_pokemon-page/css/styles.css:26]: what do you think about just writing h? 
h {
line height: 1.5;
}

Comment (ivannemeth) [reply css-basic2/css-basics_pokemon-page/css/styles.css:26]: not sure tbh, see: https://github.com/spiced-academy/chervil-web-dev/blob/main/sessions/css-basics/css-basics.md#combine-multiple-selectors-with-a-comma

Comment (emyjamalian) [reply css-basic2/css-basics_pokemon-page/css/styles.css:26]: Thanks for the reference. I've learned something :)

**Pull request merged**